tests/x11/gnome_terminal.pm

```diff
@@ -20,6 +20,7 @@ use testapi;
 sub run {
     my ($self) = @_;
     mouse_hide(1);
+    ensure_installed('gnome-terminal') if is_sle("16+") || is_leap("16+");
```

Comment:
If the gnome_terminal module should run on tumbleweed, we need to install there too (is_sle and is_leap both do return false for tumbleweed)

Maybe change to (dummy code):
ensure_installed() unless is_sle < 16 or is_leap < 16 ?

Reply:
Right, we'll keep it until there's the SR to have it installed by default

Reply:
you mean https://build.opensuse.org/requests/1255689 ? :)

Reply:
> you mean https://build.opensuse.org/requests/1255689 ? :)

Yay! Then lets add to this PR the modification to the schedule too